src/Courses/Browse.jsx

```diff
@@ -8,8 +8,10 @@ class BrowseCourses extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (prevProps != this.props) {
-      $('[data-toggle="tooltip"]').tooltip();
+    if (process.env.NODE_ENV === "production") {
```

Comment:
Why do you need to check for production / development environment here?

Reply:
uh yeah, good point, that was just to get the demo working because i didn't have bootstrap and jquery loaded, but i added them and removed this check.